locale/en.json

```diff
@@ -162,7 +162,7 @@
 		"identity-block.password-placeholder": "**********",
 		"identity-block.password-requirements": "Please enter a valid password.",
 		"identity-block.password-requirements-characters": "Must be at least %{requirementCount} total characters.",
-		"identity-block.password-requirements-lowercase": "Must have at least %{requirementCount} letter(s).",
+		"identity-block.password-requirements-lowercase": "Must have at least %{requirementCount} lowercase character(s).",
```

Comment:
Changing just this file won't update the text in the block. You'll need to run `npm run generate:intl` as well. Also make sure that the string in Lokalise is up to date, otherwise this update will get overwritten when we do updates to the locale strings.